rust/BUILD.md

```diff
@@ -0,0 +1,134 @@
+# `evmrs` Development Instructions
+
+## Requirements
+
+- install Rust toolchain >= 1.81.0 from [here](https://rustup.rs/)
+
+## Building evmrs (shared c library)
+
+- build `evmrs` in release mode
+    ```sh
+    cargo build --release # OR run `make` or `make tosca-rust` in Tosca directory
+    ```
+- build `evmrs` with debug symbols
+    ```sh
+    cargo build --profile profiling
```

Comment:
why not just cargo build? 
we may need a different explanation, is this release with symbols?